scripts/update-esbuild-versions.js

```diff
@@ -0,0 +1,81 @@
+const https = require("https");
+const { exec } = require('shelljs');
+const { mkdirSync, rmdirSync, createWriteStream } = require('fs');
+const { join } = require('path');
+const { tmpdir } = require('os');
+
+const PLATFORMS = {
+  "esbuild_darwin": "esbuild-darwin-64",
+  "esbuild_windows": "esbuild-windows-64",
+  "esbuild_linux": "esbuild-linux-64"
+}
+
+function getUrlAsString(url) {
+  return new Promise((resolve, reject) => {
+    https.get(url, (res) => {
+      if (res.statusCode !== 200) {
+        console.error(res);
+        return reject();
+      }
+      
+      let body = '';
+      res.on("data", (chunk) => body += chunk);
+      res.on("end", () => resolve(String(body)));
+    });
+  });
+}
+
+function downloadFile(url, dest) {
+  return new Promise((resolve, reject) => {
+    const file = createWriteStream(dest);
+
+    const request = https.get(url, (response) => {
+        response.pipe(file);
+    });
+
+    file.on('finish', () => {
+      file.end();
+      resolve();
+    });
+  });
+};
+
+async function main() {
+  console.log('""" Generated code; do not edit\nUpdate by running yarn update-esbuild-versions\n\nHelper macro for fetching esbuild versions for internal tests and examples in rules_nodejs\n"""\n');
```

Comment:
Perhaps preference, I like to add all the strings to an array, then join that array a output the result, rather than making multiple calls to `console.log`, eg:

```
const content = [];

content.push(''"""Generated code; do not edit');
...
content.join('\n');
```